pynvjitlink/api.py

```diff
@@ -16,6 +16,10 @@ class InputType(Enum):
     LIBRARY = 6
 
 
+def nvjitlink_version():
```

Comment:
I'm open to naming suggestions if something else seems better.

Reply:
I think this name is already good!